src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const bigReduction = 50;
+  const smallReduction = 20;
+  const price = 40;
+  let result = days * price;
+
+  if (days >= 7) {
+    result = result - bigReduction;
+  } else if (days >= 3) {
+    result = result - smallReduction;
+  }
```

Comment:
You can simplify the conditional statements by using `if` with `return` to eliminate unnecessary `else` statements. For example, you can directly return the result after applying the reduction for `days >= 7` and `days >= 3`.